test/migration/verify/verify_migration_test.go

```diff
@@ -24,7 +24,7 @@ import (
 
 func TestAfterMigration(t *testing.T) {
 	// given
-	awaitilities := WaitForDeployments(t)
+	awaitilities := WaitForOperators(t)
```

Comment:
:thinking: 
this should be called rather in the setup suite when we deploy the operators from master, and not in the "verify" one when the operators are already updated with the modified version

Reply:
Good catch. Yeah... we shouldn't verifying webhooks and other "optional" things before we update the operator. And do the full verification of the installations of all components including webhooks after we update the operator to the latest version from the paired PRs.

Reply:
I actually thought we wanted to skip these verifications for migration altogether since it's covered by the other tests anyways. So I actually missed switching the WaitForDeployments()  to WaitForOperators() in TestSetupMigration. But I think what you're suggesting would work well too. So if I understand correctly 3d7cde9fb79268d455d5394d4090a819001f9b44 is what you're expecting?